### PR TITLE
Update creating-global-styles.md

### LINKED ISSUE #7447: [doc] Improve the doc about how to handle global styles
As discussed [here](https://github.com/gatsbyjs/gatsby/issues/7413#issuecomment-413944617) we should improve the documentation about how to handle global styles and css-in-js in gatsby-browser.js .

As Kyle said: _gatsby-browser.js isn't included when server rendering which means styles there aren't extracted and inlined in the generated html files_ so we should be more clear about this.

Thanks for the report @alexandernanberg :)

### PR DESCRIPTION
**Pull request by erinfox**
Added new wording around importing global styles based on [this comment](https://github.com/gatsbyjs/gatsby/issues/7413#issuecomment-413944617) 

Closes #7447

### CODE REVIEW

File: docs/docs/creating-global-styles.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ a {
 
 Then, include the stylesheet in your site's `gatsby-browser.js` file.
 
+> _Note: This solution works when including css as those styles are extracted when building the js but not for css-in-js.
```
Comment (k3rn31p4nic) [inline docs/docs/creating-global-styles.md]: You're starting the `_` for emphasis, but you're not ending it. And therefore, it's not being rendered properly (as seen in the screenshot below). Maybe, you forgot to end it with `_`?
image

**Comment by KyleAMathews** [comment]
Deploy preview for *using-contentful* failed.

Built with commit 2961db256e8f0fb1a402f36e1a8e929c565f3e30

https://app.netlify.com/sites/using-contentful/deploys/5b856a8bb13fb12f1d553e0a

**Comment by m-allanson** [comment]
Nice, thanks!

**Comment by parikshitgupta1** [comment]
**Hey, @gatsbybot @gatsbot please add me to the organization.**

**Pull request merged**